src/tree_builder/mod.rs

```diff
@@ -67,17 +67,25 @@ pub struct TreeBuilderOpts {
     /// **Warning**: This may produce extremely incorrect results
     /// on some documents!
     pub ignore_missing_rules: bool,
+
+    /// Initial form element override. The HTML fragment parsing
+    /// algorithm uses this. Default: None
+    pub initial_form_elem: Option<Handle>,
+
+    /// The context element for the fragment parsing algorithm. Default: None
+    pub context_elem: Option<Handle>,
```

Comment:
I'm not thrilled about having `Handle`s in the `TreeBuilderOpts`, which is otherwise plain old configuration data.  (For example, we need to remember to trace them on GC.)

Can we do this another way, e.g. `TreeBuilder::new_for_fragment()`?